### ISSUE TITLE
Forms instrumentation failed

### ISSUE THREAD

**Issue by Kakifrucht** [File5/intellij-idea-guidesigner-plugin#2]
Hey there, thanks for porting this.
Unfortunately for some reason javac2 doesn't read third party libraries and I get the error that the `Forms instrumentation failed for My.form: Class not found: com.github.weisj.darklaf.components.loading.LoadingIndicator`, where LoadingIndicator class comes from darklaf. I checked that the formsClasspath in Gradle contains all directories to all libraries, which is the case.

Compiling via IntelliJ works fine.

**Comment by File5** [comment]
Hello!
Yes, it looks like there's a problem with third party libraries. Actually, if I got it right, the same problem is present even for `com.intellij.uiDesigner.core.Spacer` (components `HSpacer` and `VSpacer` in the IntelliJ IDEA GUI Designer).
Yes, it looks like the problem is connected to the classpath even though formsClasspath looks valid.

I've spent some time looking into it and I've also looked into the [old Maven plugin](https://mvnrepository.com/artifact/org.codehaus.mojo/ideauidesigner-maven-plugin). It looks like they called `setClasspath` on `com.intellij.ant.Javac2` task (Javac2Mojo.java:159). Calling that in the plugin code can probably help to fix it, although I'm not sure.

I'll look into it later when I have more time. If anyone is proficient in Gradle and Ant - any advice is appreciated!
